### DISCUSSION TITLE
Ability to edit Series tag name

### DISCUSSION THREAD

**Discussion by JarrodMiller** [advplyr/audiobookshelf#1629] [Ideas]
I have a large library of audiobooks where the sequence was detected correctly, but the series name needs corrected. Modifying these one at a time is a massive timesync. The ability to do various kinds of mass edits to a series such as:

Modify Series Name
Mass change tags for all audiobooks in the series (specifying author, publisher etc.)

**Comment by hobesman** [comment]
Sounds like something that would make sense on a series page. I made a request for a series page where aspects of the series can be edited and added to. https://github.com/advplyr/audiobookshelf/issues/711
